Feature: Document how to determine a user ID (#6689)
mattermost/docs: source/configure/user-management-configuration-settings.rst

| **Note**: You can search for users by partial first name, last name, nickname, or username.                                 |
+---------------------------------------------------------------+-------------------------------------------------------------+

> Determining a User's ID

Reviewer: ```suggestion
Identify a User's ID
```

> ---------------------

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Users can be specified in Mattermost by username or user ID. Usernames automatically resolve when a match is detected.

> Below are two ways a User's ID can be determined.

Reviewer: ```suggestion
System admins can identify a user's ID using the Mattermost API or mmctl. 
```

Using the API

> ~~~~~~~~~~~~~

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^
```

> Yes, it is possible to find a user's ID using the Mattermost API. You can use this method if you need to automate user-related tasks or integrate with external systems.

Reviewer: ```suggestion
Use this method if you need to automate user-related tasks or integrate with external systems.
```

Make an HTTP GET request to the following endpoint: ``https://your-mattermost-url/api/v4/users/username/username_here``.

> Replace **your-mattermost-url** with the URL of your Mattermost instance and **username_here** with the actual username of the user you are looking for.

Reviewer: ```suggestion
Replace ``your-mattermost-url`` with the URL of your Mattermost instance and ``username_here`` with the username you are looking for.
```

> The API response will contain a JSON object that includes the user's ID among other details.

Reviewer: ```suggestion
The API response contains a JSON object that includes the user's ID among other details.
```

Using mmctl

> ~~~~~~~~~~~

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^
```

If you prefer command-line tools, Mattermost offers mmctl for system administration.

> Open your terminal or command prompt.

Reviewer: ```suggestion
In a terminal window, use the following command to list all users and their IDs: ``mmctl user list`` to return a list of user IDs.
```

> Use the following command to list all users and their IDs: ``mmctl user list``.

Reviewer: This text has been incorporated into the update directly above. Please remove.

Author: @cwarnermm - Removed.

> You will receive a list of users with their IDs, making it easy to identify the user you're interested in.

Reviewer: This text has been incorporated into the update directly above. Please remove.

Author: @cwarnermm - Removed.
